### DISCUSSION TITLE
Hana bank app

### DISCUSSION THREAD

**Discussion by Woojoo81** [List-KR/community#18] [Q&A]
Hello, I am a user who is very satisfied with the Adguard app. I usually use Hana Bank's app, but I haven't been able to use it because the app has recently been blocked. Please make sure that the banking app is not blocked by proper verification

**Comment by piquark6046** [comment]
Do you use AdGuard DNS server with AdGuard application?
If so, please use Cloudflare DNS server instead of AdGuard DNS server temporary.

**Comment by piquark6046** [comment]
Please refer https://github.com/List-KR/List-KR/discussions/732#discussioncomment-6292291.
